src/psyclone/psyir/transformations/omp_loop_trans.py

```diff
@@ -302,14 +302,17 @@ def apply(self, node, options=None,
         :param node: the supplied node to which we will apply the \
                      OMPLoopTrans transformation
         :type node: :py:class:`psyclone.psyir.nodes.Node`
+        :param bool reprod: indicating whether reproducible reductions should
+            be used. By default the value from the config file will be used.
         :param options: a dictionary with options for transformations
                         and validation.
         :type options: Optional[Dict[str, Any]]
-        :param bool options["reprod"]:
-                indicating whether reproducible reductions should be used. \
+        :param bool reprod:
+                whether reproducible reductions should be used. \
```

Comment:
No need for line-continuation char here.

Reply:
Removed.

Reply:
Removed throughout the file.